code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -250,27 +250,25 @@
 		to_chat(M, "<span class='warning'>You can't put [p_them()] out with just your bare hands!</span>")
 		return
 
-	if(health >= 0 && !(has_trait(TRAIT_FAKEDEATH)))
-
-		if(!(mobility_flags & MOBILITY_STAND))
-			if(buckled)
-				to_chat(M, "<span class='warning'>You need to unbuckle [src] first to do that!")
-				return
-			M.visible_message("<span class='notice'>[M] shakes [src] trying to get [p_them()] up!</span>", \
-							"<span class='notice'>You shake [src] trying to get [p_them()] up!</span>")
-		else
-			M.visible_message("<span class='notice'>[M] hugs [src] to make [p_them()] feel better!</span>", \
-						"<span class='notice'>You hug [src] to make [p_them()] feel better!</span>")
-			SEND_SIGNAL(src, COMSIG_ADD_MOOD_EVENT, "hug", /datum/mood_event/hug)
-		AdjustStun(-60)
-		AdjustKnockdown(-60)
-		AdjustUnconscious(-60)
-		AdjustSleeping(-100)
-		AdjustParalyzed(-60)
-		AdjustImmobilized(-60)
-		set_resting(FALSE)
-
-		playsound(loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+	if(!(mobility_flags & MOBILITY_STAND))
+		if(buckled)
+			to_chat(M, "<span class='warning'>You need to unbuckle [src] first to do that!")
+			return
+		M.visible_message("<span class='notice'>[M] shakes [src] trying to get [p_them()] up!</span>", \
+						"<span class='notice'>You shake [src] trying to get [p_them()] up!</span>")
+	else
+		M.visible_message("<span class='notice'>[M] hugs [src] to make [p_them()] feel better!</span>", \
+					"<span class='notice'>You hug [src] to make [p_them()] feel better!</span>")
+		SEND_SIGNAL(src, COMSIG_ADD_MOOD_EVENT, "hug", /datum/mood_event/hug)
+	AdjustStun(-60)
+	AdjustKnockdown(-60)
+	AdjustUnconscious(-60)
+	AdjustSleeping(-100)
+	AdjustParalyzed(-60)
+	AdjustImmobilized(-60)

```

Comment:
i never really looked at what shaking did, i feel this may be a little powerful tbh